portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,87 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Noelle!</h1>");
+//   @Override
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     response.setContentType("text/html;");
+//     response.getWriter().println("Hello Noelle!");
+//   }
+        private final List<String> commStream = new ArrayList<>();
+
+		@Override
+		public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+			Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Can you make changes to these names, for example "Task", so that it's more relevant to your program? In your program you have Comment/Post. if you do make changes, make sure to double check that it's consistent throughout the codes.

Reply:
done! Will be visible in the next commit!